graphql/typeDefs.ts

```diff
@@ -5,6 +5,7 @@ export default gql`
     lessons: [Lesson!]!
     session: Session!
     allUsers: [User]
+    allModules: [Module]
```

Comment:
should this be `[Module]!` so we guarantee returning an array always?

Also, I think `modules` is better than `allModules`. `allUsers` should also be changed to `users` but that's for another time